src/solver/variable/economy/shortTermStorage.h

```diff
@@ -96,6 +96,14 @@ struct VCardShortTermStorage
         {
             return Antares::Data::ShortTermStorage::getVariableCaptionFromColumnIndex(indx);
         }
+
+        static std::string Unit(const unsigned int indx)
+        {
+            if (indx % 3 == 2)
+                return VCardShortTermStorage::Unit();
+            else
+                return "MW";
```

Comment:
This looks like a hack. It would be better to provide `indx` to `VCardShortTermStorage::Unit()`, right ?